functionalTest/test/pageobjects/page.js

```diff
@@ -0,0 +1,14 @@
+/**
+* main page object containing all methods, selectors and functionality
+* that is shared across all page objects
+*/
+module.exports = class Page {
+    /**
+    * Opens a sub page of the page
+    * @param path path of the sub page (e.g. /path/to/page.html)
```

Comment:
This doc is incorrect - the open() function doesn't take a parameter.

Reply:
Removed the comment.